### ISSUE TITLE
load_sample downloads file again after deleting .tar.gz

### ISSUE THREAD

**Issue by chrishavlin** [yt-project/yt#3097]
### Bug report

If you use `ds.load_sample()`, it fetches and unpacks a `.tar.gz` for the sample. If you then delete the `.tar.gz` (keeping the untarred folder) and call `ds.load_sample()` it will download and unpack again. 

**Code for reproduction**

Assuming `snapshot_003.tar.gz` is not in your yt data directory:

```python
import yt
import os  

ds = yt.load_sample("snapshot_033") # pooch fetches snapshot_033.tar.gz, unpacks to snapshot_033.tar.gz.untar

# remove the .tar.gz
os.remove(os.path.join(yt.config.ytcfg.get("yt", "test_data_dir"),'snapshot_033.tar.gz'))

# load sample again, will download and unpack again (even though the untar folder is still there) 
ds = yt.load_sample("snapshot_033")
```

**Expected outcome**

Presumably we should check for the untarred folder as well before downloading again? Easy enough to fix (I'm happy to put that fix in) but wanted to raise the issue in case there is some reason for this behavior that I'm overlooking?

**Comment by neutrinoceros** [comment]
Good news, this is solved in #3089

**Comment by neutrinoceros** [comment]
The fix was merged ! the new implementation for `load_sample` is far from perfect _but_ we can strike this one off our list !
